config/coreboot-novacustom-v560tu.config

```diff
@@ -0,0 +1,961 @@
+# CONFIG_VENDOR_SAPPHIRE is not set
+# CONFIG_VENDOR_SIEMENS is not set
+# CONFIG_VENDOR_SIFIVE is not set
+# CONFIG_VENDOR_STARLABS is not set
+# CONFIG_VENDOR_SUPERMICRO is not set
+# CONFIG_VENDOR_SYSTEM76 is not set
+# CONFIG_VENDOR_TI is not set
+# CONFIG_VENDOR_UP is not set
+CONFIG_MAINBOARD_FAMILY="Not Applicable"
+CONFIG_MAINBOARD_PART_NUMBER="V54x_6x_TU"
+CONFIG_MAINBOARD_VERSION="V560TU"
+CONFIG_MAINBOARD_DIR="clevo/mtl-h"
+CONFIG_DIMM_MAX=4
+CONFIG_DIMM_SPD_SIZE=1024
+CONFIG_FMDFILE=""
+# CONFIG_NO_POST is not set
+CONFIG_MAINBOARD_VENDOR="Clevo"
+CONFIG_CBFS_SIZE=0x1000000
+CONFIG_CONSOLE_SERIAL=y
+CONFIG_MAX_CPUS=22
+CONFIG_ONBOARD_VGA_IS_PRIMARY=y
+# CONFIG_POST_DEVICE is not set
+# CONFIG_POST_IO is not set
+CONFIG_UART_FOR_CONSOLE=0
+CONFIG_DEVICETREE="devicetree.cb"
+# CONFIG_VBOOT is not set
+CONFIG_VBOOT_VBNV_OFFSET=0x28
+CONFIG_VARIANT_DIR="igpu"
+CONFIG_OVERRIDE_DEVICETREE="variants/$(CONFIG_VARIANT_DIR)/overridetree.cb"
+# CONFIG_VGA_BIOS is not set
+CONFIG_MAINBOARD_SMBIOS_MANUFACTURER="Notebook"
+CONFIG_INTEL_GMA_VBT_FILE="src/mainboard/$(MAINBOARDDIR)/variants/$(VARIANT_DIR)/data.vbt"
+# CONFIG_DISABLE_HECI1_AT_PRE_BOOT is not set
+CONFIG_PRERAM_CBMEM_CONSOLE_SIZE=0x2000
+CONFIG_CMOS_DEFAULT_FILE="src/mainboard/$(MAINBOARDDIR)/cmos.default"
+CONFIG_CMOS_LAYOUT_FILE="src/mainboard/$(MAINBOARDDIR)/cmos.layout"
+CONFIG_BOOT_DEVICE_SPI_FLASH_BUS=0
+
+#
+# Alder Lake P
+#
+# CONFIG_BOARD_CLEVO_NS50PU is not set
+# CONFIG_BOARD_CLEVO_NV40PZ is not set
+
+#
+# Comet Lake U
+#
+# CONFIG_BOARD_CLEVO_L140CU is not set
+
+#
+# Kaby Lake U
+#
+# CONFIG_BOARD_CLEVO_N130WU is not set
+
+#
+# Meteor Lake H
+#
+# CONFIG_BOARD_CLEVO_V540TNX is not set
+# CONFIG_BOARD_CLEVO_V560TNX is not set
+# CONFIG_BOARD_CLEVO_V540TU is not set
+CONFIG_BOARD_CLEVO_V560TU=y
+
+#
+# Tiger Lake U
+#
+# CONFIG_BOARD_CLEVO_L140MU is not set
+# CONFIG_BOARD_CLEVO_NV40MZ is not set
+# CONFIG_BOARD_CLEVO_NS50MU is not set
+CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="V54x_6x_TU"
+# CONFIG_CONSOLE_POST is not set
+# CONFIG_USE_PM_ACPI_TIMER is not set
+CONFIG_TPM_PIRQ=0x61
+# CONFIG_SOC_INTEL_CSE_SEND_EOP_EARLY is not set
+CONFIG_VBOOT_FWID_VERSION="$(CONFIG_LOCALVERSION)"
+CONFIG_EC_SYSTEM76_EC_BAT_THRESHOLDS=y
+CONFIG_BOARD_CLEVO_MTLH_COMMON=y
+CONFIG_BOARD_CLEVO_V5X0TU_BASE=y
+CONFIG_EC_SYSTEM76_EC_FLASH_SIZE=0x40000
+CONFIG_ECAM_MMCONF_BASE_ADDRESS=0xc0000000
+CONFIG_ECAM_MMCONF_BUS_NUMBER=256
+CONFIG_MEMLAYOUT_LD_FILE="src/arch/x86/memlayout.ld"
+CONFIG_DCACHE_RAM_BASE=0xfef00000
+CONFIG_DCACHE_RAM_SIZE=0xc0000
+CONFIG_C_ENV_BOOTBLOCK_SIZE=0x40000
+CONFIG_DCACHE_BSP_STACK_SIZE=0x80400
+CONFIG_MAX_ACPI_TABLE_SIZE_KB=144
+CONFIG_HAVE_INTEL_FIRMWARE=y
+CONFIG_MRC_SETTINGS_CACHE_SIZE=0x10000
+CONFIG_DRIVERS_INTEL_WIFI=y
+CONFIG_IFD_BIN_PATH="3rdparty/dasharo-blobs/novacustom/v5x0tu/descriptor.bin"
+CONFIG_ME_BIN_PATH="3rdparty/dasharo-blobs/novacustom/v5x0tu/me.bin"
+CONFIG_GBE_BIN_PATH="3rdparty/dasharo-blobs/novacustom/v5x0tu/gbe.bin"
```

Comment:
mac address? DE:AD:C0:FF:EE?

Reply:
It's a clean NVM from Intel, the default MAC is 88:88:88:88:87:88. It's expected that this region is not flashed during regular firmware deployment or update.

If someone accidentally overwrites or wipes their GbE region, they can recover their mac using https://github.com/dasharo/dcu?tab=readme-ov-file#usage